build.sh

```diff
@@ -57,6 +57,24 @@ do
   esac
 done
 
+if [ "$DO_UPDATE" == true ] && [ "$QUIET" == false ]; then
```

Comment:
I don't think we want to skip the whole check on $QUIET; we just want to skip the "Checking if html-build is up to date..." line